### ISSUE TITLE
Sources & Sinks (4)

### ISSUE THREAD

**Issue by dominiklohmann** [tenzir/public-roadmap#55]
In parallel to the development of Pipeline Execution #18, we want to implement support for multiple sources and sinks as part of the `from`, `read`, `to`, and `write` operators. This roadmap item tracks priority group 4/4.

```[tasklist]
### Sources
- [ ] TCP Loader (4)
- [ ] UDP Loader (4)
- [ ] NetFlow v5/v9 & IPFIX Parser (4)
```

```[tasklist]
### Sinks
- [ ] UDP Dumper (4)
- [ ] TCP Dumper (4)
```

**Comment by mavam** [comment]
Superseded by the individual roadmap items #58 and #59.
